data/account.go

```diff
@@ -12,14 +12,15 @@ type AccountInfo struct {
 	Nonce                    uint64         `json:"nonce,omitempty"`
 	Balance                  string         `json:"balance"`
 	BalanceNum               float64        `json:"balanceNum"`
-	TokenIdentifier          string         `json:"token,omitempty"`
+	Token                    string         `json:"token,omitempty"`
```

Comment:
I think these naming are missleading. refer to `tokenName` and `tokenIdentifier` ?

Reply:
I see that you use the `token` field for the token identifier, and the `tokenIdentifier` for concatenating the token identifier with the nonce. this is wrong. rename, please

Reply:
Right, `TokenName` and `TokenIdentifier` are more appropriate.

Reply:
done.